src/test/java/org/gridsuite/study/server/SpreadsheetConfigCollectionTest.java

```diff
@@ -226,6 +256,98 @@ void testGetSpreadsheetCollectionNotFound(final MockWebServer server) throws Exc
         assertTrue(requests.stream().anyMatch(r -> r.equals("/v1/spreadsheet-config-collections/" + SPREADSHEET_CONFIG_COLLECTION_UUID_STRING)));
     }
 
+    @Test
+    void testSetSpreadsheetConfigCollection(final MockWebServer server) throws Exception {
+        // Create a study with an existing spreadsheet config collection
+        StudyEntity studyEntity = insertDummyStudy(UUID.fromString(NETWORK_UUID_STRING), CASE_LOADFLOW_UUID, SPREADSHEET_CONFIG_COLLECTION_UUID);
+        UUID studyUuid = studyEntity.getId();
+
+        // Test setting a new spreadsheet config collection with body
+        mockMvc.perform(org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post("/v1/studies/{studyUuid}/spreadsheet-config-collection", studyUuid)
```

Comment:
Add static import for post : 

import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;

and use just 'post' here ...